src/components/InformationCard.js

```diff
@@ -50,8 +50,8 @@ const InformationCard = ({
   histogramContent,
   isFiltering,
   isPrivatePolicy,
-  protocol,
-  protocolName,
+  protocolProperties,
```

Comment:
maybe use `sampleUnitProperties` for consistency if we are switching `protocolName` to` sampleUnit`